lesson28/registration_page.py

```diff
@@ -0,0 +1,42 @@
+from selenium.webdriver.common.by import By
+from datetime import datetime
+
+password = 'Qwerty123'
```

Comment:
password must not be stored in this open way
you can add hidden file in the repo (add it to gitignore) and read from it the user data